disruption_generator/cli.py

```diff
@@ -1,18 +1,70 @@
 # -*- coding: utf-8 -*-
 
 """Console script for disruption_generator."""
+import asyncio
+import asyncssh
 import sys
 import click
+import logging
+
+from . import __version__
+from .parsers.experiment_parser import ExperimentParser
+from .listener.alistener import Alistener
+from .trigger.trigger import Trigger
+from os import walk, path
+
+logger = logging.getLogger(__name__)
 
 
 @click.command()
-def main(args=None):
+@click.option(
+    "--experiments-path",
+    "-e",
+    type=click.Path(exists=True, file_okay=False, readable=True, resolve_path=True),
+    help="Path to experiments yamls",
+    default="./experiments/",
+)
+@click.version_option(version=__version__)
+def main(experiments_path):
     """Console script for disruption_generator."""
-    click.echo(
-        "Replace this message by putting your code into "
-        "disruption_generator.cli.main"
-    )
-    click.echo("See click documentation at http://click.pocoo.org/")
+    click.echo("!!! DISRUPTION AS A SERVICE !!!")
+    click.echo("!!!    USE WITH CAUTION     !!!")
+    try:
+        loop = asyncio.get_event_loop()
+        loop.run_until_complete(execute(experiments_path))
+    except (OSError, asyncssh.Error) as exc:
+        sys.exit("SSH connection failed: " + str(exc))
+
+
+async def execute(experiments_path):
+    _files = []
+    for (dirpath, dirnames, filenames) in walk(experiments_path):
+        for file in filenames:
+            _files.append(path.join(dirpath, file))
+        break  # Get's only root level directory files
+    _scenarios = []
+    for _file in _files:
+        _parser = ExperimentParser(yaml_path=_file)
+        scenario = _parser.parse()
+        _scenarios.extend(scenario)
+    for scenario in _scenarios:
+        click.echo("Scenario: %s" % scenario)
+        alistener = Alistener(scenario.listener.target)
```

Comment:
I have to say that is's a bit confusing that `scenario` variable (which holds `Disruption` object in this case) has `listener` attribute which is something completely different than `Alistener`. `Listener` class just holds some text data so I suppose it could be called differently.